apps/prax-marketing-site/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "minifront",
+  "name": "prax-marketing-website",
```

Comment:
Updated the package name to minimize confusion; I think the new one is correct, override me if not.